backend/src/modules/sendEmails/application/SenEmailCommandHandler.ts

```diff
@@ -0,0 +1,18 @@
+import { CommandResult } from '../../../shared/Module/core/application/command/CommandResult';
+import { CommandHandler } from '../../../shared/Module/core/application/command/CommandHandler';
+import { SendEmail } from './SendEmailCommand';
+import { EmailSender } from './EmailSender';
+
+export class SendEmailCommandHandler implements CommandHandler<SendEmail> {
+  constructor(private readonly emailSender: EmailSender) {}
+
+  async execute(command: SendEmail): Promise<CommandResult> {
+    await this.emailSender.sendAnEmail({
+      to: command.emailAddress,
+      subject: command.subject,
+      html: command.htmlContent,
```

Comment:
Może też nazwy zmiennych tak ładnie nazwać?
```suggestion
      emailAddress: command.emailAddress,
      subject: command.subject,
      htmlContent: command.htmlContent,
```

Reply:
👍